fuzz/crypto/types/CompactBitArray/marshalUnmarshal/main.go

```diff
@@ -0,0 +1,16 @@
+package compactunmarshal
```

Comment:
The package name and package path must be matched for working with oss-fuzz, so please change one or the other

Reply:
Could you please help me do that @cuonglm? It is 2:27AM here.

Reply:
Sure, will do it shortly

Reply:
I've actually made the update ;) Push coming up shortly